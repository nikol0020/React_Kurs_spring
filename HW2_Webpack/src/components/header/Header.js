import React from 'react';
import './Header.scss';
import Search from '../search';
import Sort from '../sort';
import '../propTypes';

const Header = () => {

    return (
        <div className="header">
            <div className="header__panel">
                <Search/>
                <Sort/>
            </div>
            <div className="header__panel-plus"></div>
        </div>
    );
};

export default Header;

