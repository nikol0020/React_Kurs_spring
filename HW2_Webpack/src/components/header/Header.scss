.header {
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 290px;
  background-color: #aff;

  &__panel{
    display: flex;
    justify-content: space-around;
    width: 100%;
    background-color: #fff;
    margin-top: 50px;
  }

}

