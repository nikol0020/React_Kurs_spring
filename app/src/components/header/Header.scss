.header {
  width: 100%;
  height: 275px;
  background-color: #a3a;
  border: 1px solid #a5a;
  position: relative;
  color: #ccc;

  &__panel{
    width: 100%;
    background-color: #a3a;
    margin-top: 60px;
  }

  &__panel-plus {
    width: 100%;
    height: 65px;
    background-color: #fff;
    position: absolute;
    bottom: 0;
  }

}

