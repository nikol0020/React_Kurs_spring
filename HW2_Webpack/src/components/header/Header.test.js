import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('<Header /> shallow rendering', () => {
    it('matches the snapshot Header', () => {
        const wrapper = shallow(<Header/>);
        expect(wrapper).toMatchSnapshot();
    });
});
