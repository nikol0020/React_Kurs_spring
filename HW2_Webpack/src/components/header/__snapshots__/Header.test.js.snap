// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> shallow rendering matches the snapshot Header 1`] = `
<div
  className="header"
>
  <div
    className="header__panel"
  >
    <Search
      data={Object {}}
    />
    <Sort
      data={Object {}}
    />
  </div>
  <div
    className="header__panel-plus"
  />
</div>
`;
